assignments/callbacks.js

```diff
@@ -19,30 +19,87 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
+
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+   return cb(arr.length);
 }
-
+getLength(items, function(length){
+  console.log(length)
+});
```

Comment:
You could forego this extra function wrapping `console.log` by just using `log` as your callback function (and this applies elsewhere in your code, too):
```javascript
getLength(items, console.log);
```
This works since `log` is just a function; specifically it is a method on the `console` object.